src/ResourceManager/Network/Commands.Network/ApplicationGateway/RedirectConfiguration/AzureApplicationGatewayRedirectConfigurationBase.cs

```diff
@@ -78,20 +78,6 @@ public PSApplicationGatewayRedirectConfiguration NewObject()
             redirectConfiguration.Name = this.Name;
             redirectConfiguration.RedirectType = this.RedirectType;
 
-            if (this.TargetUrl != null
```

Comment:
@raraveen any reason why these checks are being removed?

Reply:
@cormacpayne - I have not made these changes. The ones I have made are at https://github.com/Azure/azure-powershell/pull/7833/commits/3ad9d2ea48eef3ea90bbb0f58df630853facd096